var/spack/repos/builtin/packages/suite-sparse/package.py

```diff
@@ -78,10 +81,13 @@ def install(self, spec, prefix):
             ])
 
         # BLAS arguments require path to libraries
-        # FIXME : (blas / lapack always provide libblas and liblapack as aliases)
-        make_args.extend([
-            'BLAS=-lblas',
-            'LAPACK=-llapack'
-        ])
+        # FIXME: (blas/lapack always provide libblas and liblapack as aliases)
+        if '@4.5.1' in spec:
```

Comment:
you sure you don't need to specify `BLAS=-lblas` and `LAPACK=-llapack` for `4.5.3`?
Since you modify these lines, would you mind doing
```
to_link_flags(spec['lapack'].lapack_shared_lib)
to_link_flags(spec['blas'].blas_shared_lib)
```
instead (i.e. avoid hardcoding `lblas` and `llapack`)?